setup.py

```diff
@@ -19,10 +19,12 @@
         "synthimpute",
         "pytest",
         "pytest-dependency",
+        "pyyaml",
+        "OpenFisca-US",
         "requests",
     ],
     extras_require={
-        "dev": ["autopep8", "black", "setuptools", "wheel", "openfisca-us"]
+        "dev": ["autopep8", "black", "setuptools", "wheel", "openfisca-us"],
```

Comment:
I think it's fine to move all test-only dependencies into the `dev` requirements (`pyyaml` and `openfisca-us`).

Reply:
Just learned that PyPi is case-insensitive. Do you want me to move "pytest" and "pytest-dependency" into dev or leave in install_requires?